tests/acceptance/certificates/delete-test.js

```diff
@@ -79,6 +79,8 @@ test(`visiting ${url} and deleting a certificate should delete the certificate`,
   signInAndVisit(url);
   andThen(function() {
     let deleteButton = find('button:contains(Delete)');
+    // Clobber window confirm to accept delete.
+    window.confirm = () => { return true; };
```

Comment:
I suggest doing this globally for the whole test suite. For example as we do here: https://github.com/aptible/dashboard.aptible.com/blob/master/tests/test-helper.js#L18

Most significantly setting it here does leak it into other tests.